functions-methods.js

```diff
@@ -33,4 +69,24 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity (emailadress) {
+    const containsAt = emailadress.includes('@');
+    const containsComma = emailadress.includes(',');
+    const indexOfFinalDot = emailadress.lastIndexOf('.')
+    const containsNoDot = indexOfFinalDot !== emailadress.length -1;
+    if (containsAt && containsComma && containsNoDot) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Je zou de uitkomst meteen terug kunnen geven
```suggestion
    return containsAt && containsComma && containsNoDot;
```